.circleci/config.yml

```diff
@@ -159,15 +145,6 @@ commands:
 
 jobs:
 
-  stack_810_legacy_executable:
```

Comment:
As discussed in #1955.  I couldn't get the preprocessor stuff working for reasons mentioned there.